samples/Dantooine/Dantooine.Client/Shared/Authorization/ClaimValue.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Dantooine.BFF.Shared.Authorization
+{
+    public class ClaimValue
```

Comment:
Could this be replaced by a value tuple?

Reply:
maybe, ended up with null exceptions when I tried this.